ci-operator/jobs/openshift/gcp-pd-csi-driver-operator/openshift-gcp-pd-csi-driver-operator-master-periodics.yaml

```diff
@@ -0,0 +1,169 @@
+periodics:
+- agent: kubernetes
+  cluster: build02
+  cron: '@weekly'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: gcp-pd-csi-driver-operator
+  labels:
+    ci-operator.openshift.io/cloud: gcp
+    ci-operator.openshift.io/cloud-cluster-profile: gcp
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-gcp-pd-csi-driver-operator-master-periodic-e2e-gcp-csi-c3
+  reporter_config:
+    slack:
+      channel: '#forum-ocp-storage'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: ':red_jenkins_circle: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
+        <{{.Status.URL}}|View logs>'
```

Comment:
This is a job specific config and you create 6 jobs in this PR. You must add this chunk to all jobs. Look for `spec:` in this PR and put it between every `name:` and `spec:` here.